lib/matplotlib/mathtext.py

```diff
@@ -17,34 +17,18 @@
 
 from collections import namedtuple
 import functools
-from io import StringIO
 import logging
-import types
 
 import numpy as np
-from PIL import Image
 
 from matplotlib import (
-    _api, colors as mcolors, rcParams, _mathtext, _mathtext_data)
+    _api, rcParams, _mathtext)
```

Comment:
This is now shot enough for

`from matplotlib import _api, rcParams, _mathtext`